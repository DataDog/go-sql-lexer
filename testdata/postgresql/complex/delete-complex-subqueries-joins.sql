DELETE FROM 
  users u
USING 
  orders o,
  order_items oi,
  products p
WHERE 
  u.id = o.user_id
AND o.id = oi.order_id
AND oi.product_id = p.id
AND p.category = 'obsolete'
AND o.order_date < NOW() - INTERVAL '5 years';
