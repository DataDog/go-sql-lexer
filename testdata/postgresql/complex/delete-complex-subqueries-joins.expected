[
    {
        "expected": "DELETE FROM users u USING orders o, order_items oi, products p WHERE u.id = o.user_id AND o.id = oi.order_id AND oi.product_id = p.id AND p.category = ? AND o.order_date < NOW ( ) - INTERVAL ?",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "DELETE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]